portfolio/src/main/webapp/script.js

```diff
@@ -90,6 +85,40 @@ function filterComments() {
     const commentSection = document.getElementById('comment-list');
     // Clear current comments and re-add with appropriate filter.
     commentSection.innerHTML = '';
-    messages.forEach(message => commentSection.appendChild(createListElement(message)));
-  })
+    messages.forEach(message => commentSection.appendChild(createComment(message.message,
```

Comment:
add a newline after "appendChild(" rather than splitting up the two params. that will make it easier to read